frontend/public/storage/components/storage-overview/storage-overview.jsx

```diff
@@ -19,6 +19,8 @@ import { coFetchJSON } from '../../../co-fetch';
 const REFRESH_TIMEOUT = 5000;
 
 const CEPH_STATUS = 'ceph_health_status';
+const STORAGE_CEPH_CAPACITY_TOTAL_QUERY = 'ceph_cluster_total_bytes';
```

Comment:
@mareklibra I guess we should also use this queries for storage in cluster overview, right ?

Reply:
Right, we should be aligned with storage on that. 
We discussed these queries multiple times - @AyushAmbastha, is this the "final" metric to be used, please?

Reply:
@cloudbehl will be in a better position to answer this.

Reply:
@mareklibra I think for now this is best possible metric available from the ceph to calculate this value. As I can see in cluster dashboard we are using kubelet metric or node available.

1. Kubelet metric is not available as the cephCSI and generic CSI doesn't have a support of that right and I don't think it will be available.
2. node_filesystem_avail_bytes includes all the nodes that are present in the system and doesn't gives the actual number which can be used by customer.

I used both queries(`ceph_cluster_total_bytes` and `node_filesystem_avail_bytes`) to see what difference.

![Screenshot from 2019-04-09 13-54-27](https://user-images.githubusercontent.com/6695156/55784983-47bfe800-5acf-11e9-9d18-f2c2b9677edc.png)

if you see the screen shot:
```
ceph_cluster_total_bytes - 385568673792 = 385.5 GB
node_filesystem_avail_bytes - 5049127116800 = 5049.1GB
```

So I think we can use `ceph_cluster_total_bytes` to show the storage bytes.

Reply:
ok, thank you for exmplnation.
So I'll update ClusterOverview metric to this one for Ceph. I'll keep the default in case Ceph is not available.